pages/en/network-transition-faqs.mdx

```diff
@@ -0,0 +1,236 @@
+---
```

Comment:
@MichaelMacaulay – I think you accidentally committed this file which has the same name as the other one, except `faqs` instead of `faq`.

Reply:
You're right, my bad